public/app/plugins/datasource/influxdb/components/FluxQueryEditor.tsx

```diff
@@ -12,12 +12,10 @@ import {
   CodeEditorSuggestionItemKind,
 } from '@grafana/ui';
 import { getTemplateSrv } from '@grafana/runtime';
+import InfluxDatasource from '../datasource';
 
-interface Props {
-  target: InfluxQuery;
-  change: (target: InfluxQuery) => void;
-  refresh: () => void;
-}
+// @ts-ignore -- complicated since the datasource is not really reactified yet!
```

Comment:
This is not a real `QueryEditor` -- since influx is still an angular based datasource.

Reply:
Is there an argument to split InfluxDB & Flux into two datasource plugins? (that share some code). Since it looks pretty binary with not being able to mix flux/InfluxQL in the same data source instance anyway.

Reply:
Yes -- there is a good argument, but let's discuss